### PR TITLE
Added first route

### PR DESCRIPTION
**Pull request by danbrud**
Making a pull

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const request = require('request')
+const router = express.Router()
+
+
+router.get('/sanity', function (req, res) {
+    res.send(`Let's have some fun`)
```
Comment (Hunter-aw) [inline server/api.js:7]: this is in-sane!

**Comment by Srulda** [comment]
oh no!!!!!!

File: server/api.js (hunk 1 above)
Comment (Srulda) [reply server/api.js:7]: this is gr8

File: server/api.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const express = require('express')
+const request = require('request')
+const router = express.Router()
+
+
+router.get('/sanity', function (req, res) {
```
Comment (chenkamin) [inline server/api.js:6]: Good CoDe :)

**Pull request merged**